cluster-provision/centos8/Dockerfile

```diff
@@ -1,5 +1,5 @@
 
-FROM quay.io/fedora/fedora@sha256:38813cf0913241b7f13c7057e122f7c3cfa2e7c427dca3194f933d94612e280b
+FROM quay.io/fedora/fedora:36
```

Comment:
Isn't this risky? Fedora is known to change frequently with little assurance of stability.

Reply:
We can mirror it if needed to our quay
note that atm it blocks kubevirtci PR

Reply:
Let's split this part to separate PR and pin this down to a new hash. Keeping a floating tag can cause an issue that will be discovered later than missing sha.